tools/RecyclerChecker/RecyclerChecker.cpp

```diff
@@ -1,3 +1,8 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
 //===- RecyclerChecker.cpp ---------------------------------------------===//
```

Comment:
I think this license is from the original sample I was working off of- I don't know if it needs to be preserved since this code is considerably different than the sample. In any case, the comment following the license should be fixed :)